rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -212,10 +212,10 @@ object SystemProcesses {
             _ <- parameters match {
                   case ShortLeashParameters(codeHash, phloRate, userId, timestamp) =>
                     produce(Seq(codeHash, phloRate, userId, timestamp), ack)
```

Comment:
Probably something like

```suggestion
ShortLeashParameters(codeHash, phloRate, userId, timestamp) = parameters
_ <- produce(Seq(codeHash, phloRate, userId, timestamp), ack)
```

Also add a TODO to add a test for the case _ => below

Reply:
Actually that can be fixed even nicer, by using the case class fields instead of the match